applications/asset_tracker_v2/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -58,9 +58,6 @@ CONFIG_NET_STATISTICS_WIFI=y
 CONFIG_NET_STATISTICS_USER_API=y
 CONFIG_NET_CONTEXT_SYNC_RECV=y
 
-# Disable unused networking options
-CONFIG_NET_IPV6=n
```

Comment:
> IMO we should not without a good explanation disable IPv6 networking.

I totally agree with that, but it's not clear to me why it's been disabled, and could also be because of issues. To me it seems unrelated to this PR, and should be left out.

Reply:
`CONFIG_NET_IPV6` is enabling/disabling Zephyr's IPv6 stack. It's enabled by default for some reason while `CONFIG_NET_IPV4` is disabled by default.
So this doesn't affect on ATv2's capability to use IPv6 as we are not using Zephyr's IP stack (`CONFIG_NET_NATIVE=n`).

However, it looks like `CONFIG_NET_IPV6=y` and `CONFIG_NET_IPV4=y` are set in `prj.conf` and they've been added as part of LwM2M support so they are likely needed from that perspective to have some functions available.

From this perspective, it would make sense to remove this line here.

Reply:
Looks like you were the one that added that line @trantanen, so then I'm fine with removing it :)